src/HotChocolate/Data/src/Data/Filters/IFilterFieldDefinition.cs

```diff
@@ -0,0 +1,13 @@
+using System.Reflection;
+
+namespace HotChocolate.Data.Filters
+{
+    public interface IFilterFieldDefinition
+    {
+        MemberInfo? Member { get; set; }
+
+        IFilterFieldHandler? Handler { get; set; }
```

Comment:
this is not read-only

Reply:
:D

Reply:
I reviewed the PR and only looked for formatting issues